index.bs

```diff
@@ -609,6 +609,8 @@ This API is disabled by default in all cross-origin frames using the [[#permissi
 
 This API allows creation of an {{MLContext}} from a {{GPUDevice}} defined by WebGPU specification. See <a href="https://gpuweb.github.io/gpuweb/#security-considerations">WebGPU Security Considerations</a> for more information regarding security characteristics of this context.
 
+This API provides an abstraction across GPU, CPU and dedicated ML accelerator hardware. When using a GPU, <a href="https://www.w3.org/TR/webgpu/#security-dos">denial of service</a> considerations similar to WebGPU apply. When using a CPU or a dedicated ML accelerator, the types of potential resource contention are different and mitigations will be implementation- and configuration-dependent. Implementations should use whatever mechanisms available from the platform to prevent sites from using an unfair amount of system resources. These compute units are shared resources and the use of any compute API will affect overall performance on a fully-loaded system.
```

Comment:
The trailing dash after "implementation" looks weird. Propose: `implementation and configuration dependent"